libr/util/table.c

```diff
@@ -553,8 +553,10 @@ R_API void r_table_filter(RTable *t, int nth, int op, const char *un) {
 		if (page < 1) {
 			page = 1;
 		}
-		lrow = page_items * (page - 1);
-		uv = page_items * (page);
+		if (!ST32_MUL_OVFCHK (page, page_items)) {
```

Comment:
```suggestion
		if (!ST32_MUL_OVFCHK (page - 1, page_items)) {
```

Reply:
i dont think such chk is needed, because if page*page_items is overflowing, page-1*page_items will be overflowing too